app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "How to get into teaching without a degree"
+title: "Train to teach if you do not have a degree"
```

Comment:
I think the original title is better from an SEO perspective in terms of keyword volumes and actual queries, although I think it is slightly misleading!

Is there a better way of phrasing this that would make more sense and work for SEO?

Reply:
Can change them all to 'Get into teaching without a degree'?